frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -68,28 +68,12 @@ const register = async (
       { firstName, lastName, email },
       { withCredentials: true },
     );
-    localStorage.setItem(AUTHENTICATED_USER_KEY, JSON.stringify(data));
     return data;
   } catch (error) {
     return null;
   }
 };
 
-const setPassword = async (
```

Comment:
I think we need this flow for simply changing a user's password, not resetting it. (ie post login). Is there a reason it was removed?

Reply:
We have our own authentication infrastructure set up instead of using the default one. When resetting the password, we build our own URLs to trigger the password reset and have our own version of the oobCode. We will redo the setPassword functionality based on our version of authentication handling.